Servers/UI/OJS.Servers.Ui/ClientApp/src/components/administration/users/lecturer/LecturerInContests.tsx

```diff
@@ -75,6 +76,14 @@ const LecturerInContests = (props: ILeturerInContestsProps) => {
         },
     ] = useRemoveLecturerFromContestMutation();
 
+    useSuccessMessageEffect({
+        data: [
+            { message: addData, shouldGet: isSuccessfullyAdded },
```

Comment:
This shouldGet parameter is a bit weird? What does it relate to, what do we get if set to true?

Reply:
The ```shouldGet``` parameter determines whether a particular message should be included in the final result.